src/containers/Settings.jsx

```diff
@@ -217,9 +249,16 @@ class Settings extends React.Component {
               />
 
               <Form.Button
-                type="submit"
                 label={this.props.saveLabel || "Save Twilio Credentials"}
+                onClick={this.handleOpenTwilioDialog}
               />
+              <Dialog
+                actions={dialogActions}
+                modal={true}
+                open={this.state.twilioDialogOpen}
+              >
+                Changing Twilio credentials will break any campaigns that are currently running. Do you want to contunue?
```

Comment:
clarify that changing Account SID or Messaging Service SID, but not Auth Token, will break existing campaigns.